Index: razercfg-0.40/librazer/CMakeLists.txt
===================================================================
--- razercfg-0.40/librazer/CMakeLists.txt
+++ razercfg-0.40/librazer/CMakeLists.txt	2019-01-02 21:00:00.100364741 +0000
@@ -36,7 +36,7 @@
 
 target_link_libraries(razer usb-1.0)
 
-install(TARGETS razer DESTINATION lib)
+install(TARGETS razer DESTINATION lib${LIB_SUFFIX})
 
 if (NOT DEFINED ENV{RPM_BUILD_ROOT} AND NOT DEFINED ENV{RAZERCFG_PKG_BUILD})
 	install_exec_cmd("ldconfig"
